src/VirtoCommerce.Platform.Core/SignalR/SignalROptions.cs

```diff
@@ -0,0 +1,11 @@
+namespace VirtoCommerce.Platform.Core.SignalR
+{
+    public class SignalROptions
```

Comment:
It is absolutely bad practice to mixed the two different infrastructure per single POCO options type.
You need to define the two different options that can be defined into their own namespaces and register them independently based on the mode setting. e.g https://github.com/VirtoCommerce/vc-platform/blob/master/src/VirtoCommerce.Platform.Web/Startup.cs#L384

Reply:
Fixed